source/documentation/other-topics/decommission-unused-template-deploy-services.md

```diff
@@ -0,0 +1,72 @@
+### How to decommission unused template deploy services
+
+#### 1.	Stack Deletion (will also delete related instances, vpu’s, security groups, network gateways etc).
+
+** NOTE As a first port of call we would recommend raising a support ticket in [#ask-cloud-platform](https://mojdt.slack.com/messages/C57UPMZLY/team/U58MLFA0M/) requesting that cloud-platform delete your AWS stack. 
+
+In most cases cloud-platform’s will have the necessary credentials to delete the stack for you (most template-deploy stacks will have been created within the mojdsd AWS account).
+
+ 
+** Caution should also be taken as AWS Route53 dns configuration may be deleted (and this could be referenced in new AWS account Kubernetes setups). We would recommend making a copy of any such configuration so that it can be reinstated if necessary. 
+
+
+Should you wish to go ahead yourselves, whilst it is possible to delete from the AWS stack dashboard, for a cleaner operation (the deleting of related AWS resources) the following fabric command should be carried out:
+
+in order to proceed, you need to have :
+
+-	installed the prerequisite tools
+-	configured your AWS credentials
+-	configured your github access
+-	create your own GPG key
+
+> ##### a.	Run [mkvirtualenv](https://dsdmoj.atlassian.net/wiki/spaces/PLAT/pages/82640903/Quickstart+-+Template+Deploy) to create your local environment.
```

Comment:
This list, along with ones in the next couple sections, does not render properly ([you can check directly on github](https://github.com/ministryofjustice/cloud-platform-user-guide/blob/55aedc6a51ee23c4a0120c3aecbfa5779db78475/source/documentation/other-topics/decommission-unused-template-deploy-services.md)).

You shouldn't need `> ####` or any combination of these. Check this [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)!

Reply:
Hi @alkar
Could you please re-review the page content to see if the format is more readable.
Thanks